docs/install/installing-istio.md

```diff
@@ -84,24 +96,31 @@ mesh by [manually injecting the Istio sidecars][2].
 Enter the following command to install Istio:
 
 ```shell
-# A lighter template, with no sidecar injection.
+# A lighter template, with just pilot/gateway.
+# Based on install/kubernetes/helm/istio/values-istio-minimal.yaml
```

Comment:
I would expand that sentence a bit to " Based on the content of the ... directory"